cves/CVE-2017-15706.yml

```diff
@@ -206,4 +219,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: The biggest mistake seems to be simple not proofreading of the documentation
```

Comment:
I think you meant "simply not proofreading the documentation"

Reply:
hilarious error resolved.